var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -62,6 +63,11 @@ class Gdb(AutotoolsPackage, GNUMirrorPackage):
     # https://sourceware.org/pipermail/gdb-patches/2021-February/176622.html
     patch('importlib.patch', when="@8.3.1:10.2 ^python@3.4:")
 
+    # patch to use decltype instead of typeof to fix compile on glibc 2.35
```

Comment:
Thanks for your patch. Can you please provide some additional context on this. IE what system (OS) did you test on? Where does this patch come from? Is this a patch from? Is it going to be merged into upstream GDB? Did you confirm that it still works if you build with older gcc with some test output? I appreciate your time and contributions